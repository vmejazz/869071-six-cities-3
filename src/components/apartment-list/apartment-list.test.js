import React from "react";
import renderer from "react-test-renderer";
import ApartmentList from "./apartment-list.jsx";

const apartmentTitleArray = [
  `First title`,
  `Second title`,
  `Third title`,
  `Another title`
];

describe(`Test Apartment list`, () => {

  it(`<ApartmentList /> should render list`, () => {
    const tree = renderer
      .create(
          <ApartmentList
            apartmentTitleArray={apartmentTitleArray}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

});
