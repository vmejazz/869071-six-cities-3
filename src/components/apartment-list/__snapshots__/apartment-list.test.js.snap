// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Apartment list <ApartmentList /> should render list 1`] = `
<div
  className="cities__places-list places__list tabs__content"
>
  <ApartmentCard
    isFavoriteList={false}
    placeOffer={
      Object {
        "apartmentDetails": Array [],
        "bedrooms": 1,
        "city": "",
        "description": "",
        "id": 1,
        "maxGuests": 1,
        "ownerInfo": Object {
          "name": "",
          "srcAvatar": "",
          "super": true,
        },
        "positions": Array [
          1,
          1,
        ],
        "premium": true,
        "price": 120,
        "rate": 1,
        "imageURLs": Array [],
        "srcImg": "",
        "title": "",
        "type": "",
      }
    }
  />
</div>
`;
